### ISSUE TITLE
[BUG/ISSUE] AEIC VOC scale factors might not be right (?)

### ISSUE THREAD

**Issue by flavioquadros** [geoschem/geos-chem#915]
### Description of the problem
The scale factors applied to aviation VOC emissions were changed on v13.2.0 (https://github.com/geoschem/geos-chem/pull/776), but I am a bit unsure if the added carbon correction was needed, could someone please check this issue again? I was under the impression that the HC variable in AEIC was in CH4-equivalent grams, as mentioned in the papers describing the inventory (https://doi.org/10.1016/j.trd.2013.07.001, https://doi.org/10.1016/j.atmosenv.2011.07.012). The speciation factors seem to be those adopted by the EPA (https://www.epa.gov/regulations-emissions-vehicles-and-engines/organic-gas-speciation-profile-aircraft), which are also implemented by the FAA's emissions model (https://aedt.faa.gov/Documents/AEDT3d_UserManual.pdf, item 5.1.1.8).

In GEOS-Chem, we have factor `114` (=1.16e-3) which does (g)->(kg) and the conversion of THC (total hydrocarbons, base CH4) to TOG (total organic gases, in overall mass). And then factors `101` ... `109` do the speciation on that as per the EPA profile. So, unless I am missing something, weren't the resulting numbers prior to the patch already in total kg of each species?

### GEOS-Chem version
v13.2.x

**Comment by flavioquadros** [comment]
Thanks for the response!

The thing is I don't think these factors (101-109) assume the input is in kg C. They should be applied on total organic gases ("TOG") expressed as kg. This is, in turn, calculated from total hydrocarbons ("THC") expressed as equivalent kg of CH4, which is how turbojet emissions are measured for regulation. The hydrocarbon emissions on AEIC represent this THC in base CH4, even if at some point the species derived from it were held in GEOS-Chem as kg C, so I would think there should be no need to apply factors 40-46, 49, 83-84.

This assumption of (total) kg of organic gas is described in the EPA documents recommending this specific speciation profile (https://www.epa.gov/regulations-emissions-vehicles-and-engines/organic-gas-speciation-profile-aircraft). This is also briefly described in the technical manual of the FAA's emissions model, which adopts the same scheme (I had the wrong link in the previous message, it's items 5.1.1.8-9 of *this* pdf: https://aedt.faa.gov/Documents/AEDT3d_TechManual.pdf).

**Comment by yantosca** [comment]
Also tagging @sdeastham here who was the developer of the AEIC inventory.

**Comment by sdeastham** [comment]
Thanks @yantosca and @flavioquadros for digging into this! The original derivation of the UHC speciation factors comes from the AEDT guidance (https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.719.2090&rep=rep1&type=pdf), which itself is derived from an EPA document which can be found at https://www.epa.gov/sites/production/files/2016-09/420r09903_0.xls. Using acetone as a reference point, the factor 3.69...e-3 is shown as specifically the raw mass fraction of acetone in the exhaust. So I think @flavioquadros has it right and we _don't_ need to adjust based on the number of carbon atoms. This may even be a case of the original code being incorrect, way back in `AEIC_mod.F90` (or even beyond, in the now-ancient `AEDT_mod.F`!). I would not be surprised if we were emitting using the "true" weight fractions even though species like `ACET` were defined in kgC. Unburned hydrocarbons have generally been a minor contributor to aviation impacts so errors like this may not have been caught.

In short my instinct is that the mass fractions are likely usable without correction, but it would be worth re-deriving them to make absolutely sure. My apologies to @msulprizio who I fear I may have given the wrong guidance to, based on a historical view of the code rather than on the source data!

@flavioquadros - if you have an even more up to date speciation (since the GC mechanism has changed substantially since this was last really looked at), now might be a great time to implement it!

**Comment by yantosca** [comment]
Thanks @sdeastham and @flavioquadros.  If one of you could make a PR with the proper speciations that would be much appreciated.

**Comment by flavioquadros** [comment]
Thanks @sdeastham and @yantosca, sorry for taking so long to respond. I have looked here and indeed the speciation could use some species newly added to GEOS-Chem (like ethylene for example). I will send a PR with the fix on Monday.

**Comment by yantosca** [comment]
We can close this issue as PR #1067 has now been merged into the GEOS-Chem 13.4.0 development stream.
